src/promptflow-core/promptflow/executor/_tool_resolver.py

```diff
@@ -537,6 +554,13 @@ def _resolve_llm_node(self, node: Node, convert_input_types=False) -> ResolvedTo
         updated_node.inputs[key] = InputAssignment(value=connection, value_type=InputValueType.LITERAL)
         if convert_input_types:
             updated_node = self._convert_node_literal_input_types(updated_node, tool)
+        # Store flow inputs list as a node input to enable tools to identify these inputs,
```

Comment:
We may share the same function (maybe something like update_inputs_to_escape(updated_node))

Reply:
updated.